static/js/components/forms/validation.test.ts

```diff
@@ -69,37 +78,74 @@ describe("form validation util", () => {
     })
   })
 
-  it("produces a validation schema for fields regardless of whether they're required or not", () => {
```

Comment:
I removed this test because the comparison of `schema.toString()` just resulted in `[object Object]` being compared. This particular test didn't seem to do much. Other tests below I adjusted to run a validation

Reply:
hmm yeah definitely a weird test...I did modify it a bit and get it to pass by doing `JSON.stringify` on the two objects instead, but I had to modify the compared-to schema bunch to get that to pass too.